be/src/exec/tablet_sink.cpp

```diff
@@ -368,6 +370,10 @@ void NodeChannel::try_send_batch() {
     if (row_batch->num_rows() > 0) {
         SCOPED_ATOMIC_TIMER(&_serialize_batch_ns);
         row_batch->serialize(request.mutable_row_batch());
+        if (request.row_batch().ByteSizeLong() >= double(config::brpc_max_body_size) * 0.95f) {
```

Comment:
ByteSizeLong() will recursively calls ByteSizeLong() on all embedded messages. It is expensive.
And it this message really help for debugging? First, there is no context info in log. Second, what should we do if we meet this message?

Reply:
`row_batch()` only contains primitive types, so there is no problem of recursive calls. When it fails because the data sent is too large, it will only report reset by peer errors. It is difficult to determine the specific reason.